### PR TITLE
Source Google Analytics (GA4): Add dimensionFilter and metricFilter

### PR DESCRIPTION
**Pull request by mpetrykin**
## What
Add dimensionFilter and metricFilter parameters to custom_report_shema.json. This PR relates to issue #27119.

## How
Add dimensionFilter, metricFilter and filters' expressions described in [runReport documentation](https://developers.google.com/analytics/devguides/reporting/data/v1/rest/v1beta/properties/runReport) and in [FilterExpression documentation](https://developers.google.com/analytics/devguides/reporting/data/v1/rest/v1beta/FilterExpression) to custom_report_shema.json.

### CODE REVIEW

**Comment by mpetrykin** [comment]
@marcosmarxm hi! Can i ask you to review this?

**Review by sajarin — CHANGES_REQUESTED** [review]
Thanks for the PR and feature request! 

Could you run the format command: `./gradlew --no-daemon  :airbyte-integrations:connectors:source-google-analytics-data-api:airbytePythonFormat`

And commit the files?

**Comment by mpetrykin** [comment]
@sajarin done  :heavy_check_mark:

**Review by sajarin — APPROVED** [review]

**Comment by mpetrykin** [comment]
@grubberr, @bazarnov Hello! Could you help me with accepting the pull request, because this feature helps to save API credits, so that it will be possible to specify, for example, certain events that the API should return, and thereby reduce the amount of data returned.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/source.py [hunk 1]
```diff
@@ -461,12 +465,23 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
     def instantiate_report_class(report: dict, config: Mapping[str, Any]) -> GoogleAnalyticsDataApiBaseStream:
         cohort_spec = report.get("cohortSpec")
         pivots = report.get("pivots")
+
+        dimension_filter = report.get("dimensionFilter")
+        metric_filter = report.get("metricFilter")
```
[the remark this answers is not in the document]
Comment (marcosmarxm) [reply airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/source.py:479]: @mpetrykin did you check my latest comment?

**Pull request closed without merging**